generator/mavparse.py

```diff
@@ -257,7 +261,9 @@ def start_element(name, attrs):
                 if (value > self.enum[-1].highest_value):
                     self.enum[-1].highest_value = value
                 # append the new entry
-                self.enum[-1].entry.append(MAVEnumEntry(attrs['name'], value, '', False, autovalue, self.filename, p.CurrentLineNumber))
+                self.enum[-1].entry.append(MAVEnumEntry(attrs['name'], value, '', False, False, autovalue, self.filename, p.CurrentLineNumber))
```

Comment:
Shouldn't the `False` be `self.wip` ?

Reply:
No. At this point I don't know if the value is a wip or not. I discover that later on. I am using False as the default though because most entries are not wip.